shared/src/components/SchemaBuilder.vue

```diff
@@ -444,6 +462,22 @@ export default Vue.extend({
       }
     }
 
+    // Apply when not dragging rows so it can be transparent
+    .tabulator:not(.tabulator-block-select) .tabulator-row .tabulator-frozen {
```

Comment:
just double check this all looks good on `light` mode also, but seems to be great!